src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -287,6 +287,64 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="CopyRepoArtifacts"
```

Comment:
This target should probably have an inputs/output declaration. The input could be the RepoManifestFile and output a semaphore file.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18153/commits/9bfacadeccfd5235e7e99c2d624cb92b7807b176